src/communication/azext_communication/aaz/latest/communication/email/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
```

Comment:
Please upload your local aaz repo change and make a pull request in aaz repo.

Reply:
Thank you. https://github.com/Azure/aaz/pull/408